### PR TITLE
fix(arduino): adds ESP32-P4 to the package description

### PR DESCRIPTION
**Pull request by SuGlider**
-----------
## Description of Change
Updates the Package description to add the P4 in 3.1.0

## Tests scenarios
N/A

## Related links
N/A

### CODE REVIEW

**Comment by VojtechBartoska** [comment]
Lets merge this after P4 initial support is merged.

**Comment by lucasssvaz** [comment]
> Lets merge this after P4 initial support is merged.
> 
>     * we need to update also SoCs overview table in project documentation page

We can already merge. This is targeting the release v3.1.x branch and wont appear in the main page

**Comment by VojtechBartoska** [comment]
@lucasssvaz you are right

**Review by lucasssvaz — APPROVED** [review]

**Pull request merged**
